### ISSUE TITLE
Crash - System Exception

### ISSUE THREAD

**Issue by JaredStemper** [emoacht/Monitorian#359]
Monitorian worked perfectly on my builtin monitor + external monitors and then began to crash with exception logs. 

Please let me know if there is more information I can provide to help debug.

```
[Date: 10/4/2022 10:06:02 AM Ver: 3.13.0.0]
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. HResult: -2146233088
 ---> System.Management.ManagementException: Not found  HResult: -2146233087 ErrorInformation: NULL ErrorCode: NotFound
   at System.Management.ManagementException.ThrowWithExtendedInfo(ManagementStatus errorCode)
   at System.Management.ManagementObject.Initialize(Boolean getObject)
   at System.Management.ManagementBaseObject.get_ClassName()
   at System.Management.ManagementClass.GetInstances(EnumerationOptions options)
   at System.Management.ManagementClass.GetInstances()
   at Monitorian.Core.Models.Monitor.MonitorManager.MonitorData.GetSystem()
   at Monitorian.Core.Models.Monitor.MonitorManager.MonitorData.<PopulateAsync>d__33.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Monitorian.Core.Models.Monitor.MonitorManager.<ProbeMonitorsAsync>d__11.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Monitorian.Core.ViewModels.ProbeSectionViewModel.<>c.<<PerformProbe>b__8_0>d.MoveNext()
   --- End of inner exception stack trace ---

[Date: 10/4/2022 10:06:04 AM Ver: 3.13.0.0]
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. HResult: -2146233088
 ---> System.Management.ManagementException: Not found  HResult: -2146233087 ErrorInformation: NULL ErrorCode: NotFound
   at System.Management.ManagementException.ThrowWithExtendedInfo(ManagementStatus errorCode)
   at System.Management.ManagementObject.Initialize(Boolean getObject)
   at System.Management.ManagementBaseObject.get_ClassName()
   at System.Management.ManagementClass.GetInstances(EnumerationOptions options)
   at System.Management.ManagementClass.GetInstances()
   at Monitorian.Core.Models.Monitor.MonitorManager.MonitorData.GetSystem()
   at Monitorian.Core.Models.Monitor.MonitorManager.MonitorData.<PopulateAsync>d__33.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Monitorian.Core.Models.Monitor.MonitorManager.<ProbeMonitorsAsync>d__11.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Monitorian.Core.ViewModels.ProbeSectionViewModel.<>c.<<PerformProbe>b__8_0>d.MoveNext()
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.TaskExceptionHolder.Finalize()
```
[exception.log](https://github.com/emoacht/Monitorian/files/9708257/exception.log)

**Comment by emoacht** [comment]
Thanks for reporting.
When and how often you have encountered this exception?
In addition, what is your OS version? I have received several reports on WMI on insider preview versions to date.

**Comment by emoacht** [comment]
Fixed by 4e1e900d0d57c0a9dfe313993235c7e87470ff94
